bin/beaver-install

```diff
@@ -36,11 +43,13 @@ generate_and_build()
         beaver docker build -f "Dockerfile${DIST:+.$DIST}" "$@" .
     else
         # Build the docker image based on a beaver.Dockerfile base file
+        set -x
         beaver docker gen-dockerfile \
             -i beaver.Dockerfile \
-            -o beaver.Dockerfile.generated
+            -o beaver.Dockerfile.generated \
+            "${dockerfile_commands[@]-}"
```

Comment:
What is this `-` at the end for?

Reply:
If `dockerfile_commands` is empty, turns it into an empty string instead of an error